src/ResourceManager/Network/Commands.Network/AzureRM.Network.psd1

```diff
@@ -338,7 +338,34 @@ CmdletsToExport = 'Add-AzureRmApplicationGatewayAuthenticationCertificate',
                'Get-AzureRmServiceEndpointPolicyDefinition', 
                'Set-AzureRmServiceEndpointPolicyDefinition', 
                'Add-AzureRmServiceEndpointPolicyDefinition', 
-               'Set-AzureRmServiceEndpointPolicy'
+               'Set-AzureRmServiceEndpointPolicy',
+               'Remove-AzureRmExpressRouteCircuitConnectionConfig',
+               'New-AzureRmVirtualWan',
+			   'Set-AzureRmVirtualWan',
+               'Get-AzureRmVirtualWan',
+               'Remove-AzureRmVirtualWan',
+               'New-AzureRmVirtualHub',
+               'Get-AzureRmVirtualHub',
+               'Set-AzureRmVirtualHub',
+               'Remove-AzureRmVirtualHub',
+               'New-AzureRmVpnGateway',
+               'Get-AzureRmVpnGateway',
+               'Set-AzureRmVpnGateway',
+               'Remove-AzureRmVpnGateway',
+               'New-AzureRmVpnSite',
+               'Get-AzureRmVpnSite',
+               'Set-AzureRmVpnSite',
+               'Remove-AzureRmVpnSite',
+               'New-AzureRmVpnConnection',
+               'Get-AzureRmVpnConnection',
+               'Set-AzureRmVpnConnection',
+               'Remove-AzureRmVpnConnection',
+               'New-AzureRmHubVirtualNetworkConnection',
+               'Get-AzureRmHubVirtualNetworkConnection',
+               'Set-AzureRmHubVirtualNetworkConnection',
+               'Remove-AzureRmHubVirtualNetworkConnection',
+               'Get-AzureRmVirtualWanVpnSitesConfiguration',
+			   'Get-AzureRmVirtualWanSupportedSecurityProvider'
 
```

Comment:
Please also update Az.Network.psd1

Reply:
Updated the psd file - is there anything else need to support this?